.github/workflows/nodejs.yml

```diff
@@ -71,9 +71,9 @@ jobs:
           path: |
             node_modules
             */*/node_modules
-          key: webpack-cli-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-${{ hashFiles('**/yarn.lock', './yarn.lock') }}
+          key: b-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-${{ hashFiles('**/yarn.lock', './yarn.lock') }}
```

Comment:
```suggestion
          key: ${{ secrets.CACHE_VERSION }}-${{ runner.os }}-${{ matrix.webpack-version }}-yarn-${{ hashFiles('**/yarn.lock', './yarn.lock') }}
```

Reply:
I think we need investigate why we don't invalidate cache on windows, I think we need run `lerna bootstrap` after install deps, let's do it in separate PR

Reply:
will do